src/lib/firebase.js

```diff
@@ -5,12 +5,12 @@ import 'firebase/firestore';
 
```

Comment:
I think that at some point it would be good to take @MikeBLambert 's advice and store credentials in a `.env` file, but I just wanted to get this set up before we lost our read/write permissions - I def want to avoid getting a large bill from firestore 😄